### ISSUE TITLE
TypeError: cannot use a string pattern on a bytes-like object

### ISSUE THREAD

**Issue by mdjabc** [catboost/catboost#921]
Problem:TypeError: cannot use a string pattern on a bytes-like object
catboost version:0.16
Operating System:python 3.73
CPU:yes
# GPU:no
I use the code from https://catboost.ai/docs/concepts/python-reference_catboostclassifier_plot_tree.html, but meet the problem

TypeError                                 Traceback (most recent call last)
<ipython-input-7-0cd7e6395c3b> in <module>
     17 
     18 model.plot_tree(
---> 19     tree_idx=0,
     20     # pool=pool,
     21 )

D:\anaconda\lib\site-packages\catboost\core.py in plot_tree(self, tree_idx, pool)
   2482                     shape = 'rect'
   2483 
-> 2484                 graph.node(str(current_size), node_label, color=color, shape=shape)
   2485 
   2486                 if (current_size > 0):

D:\anaconda\lib\site-packages\graphviz\dot.py in node(self, name, label, _attributes, **attrs)
    129         """
    130         name = self._quote(name)
--> 131         attr_list = self._attr_list(label, attrs, _attributes)
    132         line = self._node % (name, attr_list)
    133         self.body.append(line)

D:\anaconda\lib\site-packages\graphviz\lang.py in attr_list(label, kwargs, attributes)
    112     ' [eggs=""]'
    113     """
--> 114     content = a_list(label, kwargs, attributes)
    115     if not content:
    116         return ''

D:\anaconda\lib\site-packages\graphviz\lang.py in a_list(label, kwargs, attributes)
     83     'label=spam eggs="" ham="ham ham"'
     84     """
---> 85     result = ['label=%s' % quote(label)] if label is not None else []
     86     if kwargs:
     87         items = ['%s=%s' % (quote(k), quote(v))

D:\anaconda\lib\site-packages\graphviz\lang.py in quote(identifier, html, valid_id, dot_keywords)
     48     '"<>"'
     49     """
---> 50     if html(identifier) and not isinstance(identifier, NoHtml):
     51         pass
     52     elif not valid_id(identifier) or identifier.lower() in dot_keywords:

TypeError: cannot use a string pattern on a bytes-like object

I will be thankful if you can give me help.

**Comment by mani2106** [comment]
I have the same problem when training for my dataset.
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-8-5fbe09a9f8bc> in <module>
----> 1 model.plot_tree(tree_idx=2)

~\AppData\Local\Continuum\anaconda3\envs\catboost\lib\site-packages\catboost\core.py in plot_tree(self, tree_idx, pool)
   2482                     shape = 'rect'
   2483 
-> 2484                 graph.node(str(current_size), node_label, color=color, shape=shape)
   2485 
   2486                 if (current_size > 0):

~\AppData\Local\Continuum\anaconda3\envs\catboost\lib\site-packages\graphviz\dot.py in node(self, name, label, _attributes, **attrs)
    129         """
    130         name = self._quote(name)
--> 131         attr_list = self._attr_list(label, attrs, _attributes)
    132         line = self._node % (name, attr_list)
    133         self.body.append(line)

~\AppData\Local\Continuum\anaconda3\envs\catboost\lib\site-packages\graphviz\lang.py in attr_list(label, kwargs, attributes)
    112     ' [eggs=""]'
    113     """
--> 114     content = a_list(label, kwargs, attributes)
    115     if not content:
    116         return ''

~\AppData\Local\Continuum\anaconda3\envs\catboost\lib\site-packages\graphviz\lang.py in a_list(label, kwargs, attributes)
     83     'label=spam eggs="" ham="ham ham"'
     84     """
---> 85     result = ['label=%s' % quote(label)] if label is not None else []
     86     if kwargs:
     87         items = ['%s=%s' % (quote(k), quote(v))

~\AppData\Local\Continuum\anaconda3\envs\catboost\lib\site-packages\graphviz\lang.py in quote(identifier, html, valid_id, dot_keywords)
     48     '"<>"'
     49     """
---> 50     if html(identifier) and not isinstance(identifier, NoHtml):
     51         pass
     52     elif not valid_id(identifier) or identifier.lower() in dot_keywords:

TypeError: cannot use a string pattern on a bytes-like object
```

catboost version: 0.16
Python: 3.6.8.
Operating System: Windows 8.1
CPU: yes
graphviz version: 0.11.1

From the [tutorial text](https://github.com/catboost/tutorials/blob/master/model_analysis/visualize_decision_trees_tutorial.ipynb)

> Currently only symmetric trees can be visualized.

So, is the problem due to an unsymmetric decision Tree in both of our cases?

**Comment by annaveronika** [comment]
This is fixed in catboost 0.16.3
